addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -859,6 +859,22 @@ export class OdooEditor extends EventTarget {
             }
         }
     }
+    /**
+     * Resets the selection to the last selection within the history if there is no selection.
+     */
+    resetLostSelection() {
+        // TODO Replace with historyResetLatestComputedSelection() when it will work.
+        if (!this._isSelectionInEditable(this.document.getSelection())) {
+            // Find the most recent history step with a selection.
+            for (let index = this._historySteps.length - 1; index >= 0; index--) {
+                const historyStep = this._historySteps[index];
+                if (historyStep && historyStep.selection && historyStep.selection.anchorNodeOid) {
+                    this.historySetSelection(historyStep);
+                    return;
+                }
+            }
+        }
+    }
```

Comment:
Could this (and its use) extracted in a separate commit so that we can validate it with the editor team? I guess the commit would make sense on its own as a fix anyway, right? I know we recently send a bug report to the editor team with the text selection being lost after any click in the editor panel... so I guess they are currently making a similar fix :grimacing: Maybe talk directly with them about this.

Reply:
Extracted to https://github.com/odoo/odoo/pull/76207